### DISCUSSION TITLE
Authentication -> Configuration -> Settings navigation not present in local installation

### DISCUSSION THREAD

**Discussion by felixguendling** [supabase/supabase#4694] [Questions] [answered]
The documentation states that I should go to : https://supabase.com/docs/guides/auth/auth-email

  - Go to your Supabase Project Dashboard
  - In the left sidebar, click the Authentication icon (near the top)
  - Click Settings from the list to go to the Authentication Settings page

This button is only available in the version on app.supabase.io but is missing when trying supabase locally (see blow).

Is there any documentation about the differences between the docker container that is available for self-hosting / local development and the version on app.supabase.io? Which features are missing in the self-hosted version and why?

![Screenshot from 2021-12-27 23-32-42](https://user-images.githubusercontent.com/1915889/147511381-ed55c6bc-4eec-428c-9d71-c504b2764cd5.png)
![Screenshot from 2021-12-27 23-32-09](https://user-images.githubusercontent.com/1915889/147511383-f71197d7-3c89-4b02-a5c2-aed642f9ef52.png)

**Comment by joeczucha** [comment, marked as answer]
According to https://github.com/supabase/supabase/discussions/4444, it sounds like the stuff under settings isn't included in the self-hosted version yet and that, when it is, it will be done through config files rather than through the dashboard.

**Reply by felixguendling** [reply]
Thank you for linking #4444 - this answers my question.

**Comment by kk21** [comment]
https://github.com/supabase/supabase/issues/4440

**Comment by clemwo** [comment]
Additional question: Does this mean Email Authentication can't be setup locally currently?  
Or is there another way to achieve this?

**Reply by spencerflagg** [reply]
For posterity...
To set up email authentication on a self-hosted Supabase instance, you need to edit this section of the `.env`:

(mailgun, for example)
```
SMTP_HOST=smtp.mailgun.org
SMTP_PORT=587
SMTP_USER=postmaster@yoursite.com
SMTP_PASS=password
SMTP_SENDER_NAME=Postmaster at Yoursite
```
